design/FY2023/NFP-Space Sizing and HVAC.md

```diff
@@ -114,48 +138,198 @@ SpaceHVAC:EquipmentConnections,
-       \type real
-       \minimum 0
-       \maximum 1.0
-       \autosizable
+   N1, \field Space 1 Maximum Air Flow Rate
        \default autosize
-   A4, \field Outlet 2 Node Name
+       \units m3/s
+       \minimum 0.0
+       \autosizable
+  A10, \field Space 2 Name
+       \required-field
+       \type object-list
+       \object-list SpaceNames
+  A11, \field Space 2 Supply Node Name
        \required-field
        \type node
-   N2, \field Outlet 2 Flow Fraction
+   N2; \field Space 2 Maximum Air Flow Rate
+       \default autosize
+       \units m3/s
+       \minimum 0.0
+       \autosizable
+```
+
+### ZoneHVAC:SpaceDistribution:NonAirFlow
+* *New object, splits the output from a single piece of zone equipment to one or more Spaces.*
+* *Would likely also add a ZoneHVAC:SpaceDistributionList that's referenced at the end of the ZoneHVAC:EquuipmentConnections object
+```
+ZoneHVAC:SpaceDistribution:NonAirFlow,
+       \extensible:2
+       \memo Distributes the output from a piece of zone equipment to one or more Spaces in the Zone.
+   A1, \field Name
+       \required-field
+       \reference SpaceDistribution
+   A2, \field Zone Equipment Object Type
+       \required-field
+       \type choice
+       \key ZoneHVAC:Baseboard:RadiantConvective:Electric
+       \key ZoneHVAC:Baseboard:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:RadiantConvective:Steam
+       \key ZoneHVAC:CoolingPanel:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:Convective:Electric
+       \key ZoneHVAC:Baseboard:Convective:Water
+       \key ZoneHVAC:HighTemperatureRadiant
+       \key ZoneHVAC:LowTemperatureRadiant:VariableFlow
+       \key ZoneHVAC:LowTemperatureRadiant:ConstantFlow
+       \key ZoneHVAC:LowTemperatureRadiant:Electric
+   A3, \field Zone Equipment Name
+       \required-field
+       \type object-list
+       \object-list ZoneEquipmentNames
+   A5, \field Thermostat Control Method
+       \type choice
+       \key SingleSpace
+       \key SpaceAverage
+       \default SpaceAverage
+   A6, \field Thermostat Space Name
+       \note Used with SingleSpace thermostat control method
+       \type object-list
+       \object-list SpaceNames
+   A7, \field Space Sizing Basis
+       \type choice
+       \default DesignLoad
+       \key DesignLoad
+       \key FloorArea
+       \key Volume
+       \key PerimeterLength
+   A8, \field Space 1 Name
+       \begin-extensible
+       \required-field
+       \type object-list
+       \object-list SpaceNames
+   N1, \field Space 1 Output Fraction
+       \default autosize
        \units dimensionless
-       \type real
-       \minimum 0
+       \minimum 0.0
        \maximum 1.0
        \autosizable
+   A9, \field Space 2 Name
+       \required-field
+       \type object-list
+       \object-list SpaceNames
+   N2, \field Space 2 Output Fraction
        \default autosize
+       \units dimensionless
+       \minimum 0.0
+       \maximum 1.0
+       \autosizable
```

Comment:
Should probably run these extensible fields out to about 10 in the idd when implemented. I can see a hallway with 5 or more spaces on a single zone on each side of the hallway.